app/genesis.go

```diff
@@ -107,6 +110,15 @@ func NewDefaultGenesisState() GenesisState {
 			sdk.MsgTypeURL(&group.MsgUpdateGroupPolicyAdmin{}),
 			sdk.MsgTypeURL(&group.MsgUpdateGroupPolicyDecisionPolicy{}),
 			sdk.MsgTypeURL(&group.MsgUpdateGroupPolicyMetadata{}),
+			sdk.MsgTypeURL(&group.MsgVote{}),
+			sdk.MsgTypeURL(&group.MsgWithdrawProposal{}),
+			sdk.MsgTypeURL(&oracletypes.MsgActivate{}),
+			sdk.MsgTypeURL(&oracletypes.MsgCreateDataSource{}),
+			sdk.MsgTypeURL(&oracletypes.MsgCreateOracleScript{}),
+			sdk.MsgTypeURL(&oracletypes.MsgEditDataSource{}),
+			sdk.MsgTypeURL(&oracletypes.MsgEditOracleScript{}),
+			sdk.MsgTypeURL(&oracletypes.MsgReportData{}),
+			sdk.MsgTypeURL(&oracletypes.MsgRequestData{}),
```

Comment:
Is there a possible way to declare allow messages just one place and use both on default genesis + upgrade handler?

Reply:
let's do it in the next PR. (https://github.com/bandprotocol/chain/pull/304). As that PR, move the upgrade handler to be in individual files.